### DISCUSSION TITLE
proximity alerts

### DISCUSSION THREAD

**Discussion by PetOnGitHub** [osmandapp/OsmAnd#12966] [Q&A]
Does anyone know if there is a way to turn on only proximity alerts without navigation?
Or to turn off the voice during navigation to reach the same goal?

**Comment by Lee-Carre** [comment]
Related: #12987.

> proximity alerts

For PoIs? You might try the PoI overlay under `configure map`.

> turn off the voice during navigation

When configuring your navigation (e.g. setting destination & profile to use), there might be a toggle button already (speaker icon), else it'll be in the options menu.
